serverless.yml

```diff
@@ -0,0 +1,112 @@
+
+
+service: users-api
+
+provider:
+  name: aws
+  runtime: nodejs10.x
+  iamRoleStatements:
+    - Effect: Allow
+      Action:
+        - dynamodb:Query
+        - dynamodb:GetItem
+        - dynamodb:PutItem
+        - dynamodb:UpdateItem
+        - dynamodb:DeleteItem
+      Resource:
+        - arn:aws:dynamodb:${opt:region, self:provider.region}:*:table/users
+
+
+package:
+  exclude:
```

Comment:
> Excellent. Great to see you are removing unnecessary overhead to lighten the Lambdas.

@afaq-sf Thank you.  By default serverless will not include dev dependencies which is nice.  I can also make aws-sdk a dev dependency because lambda includes those in the environment automatically.  The unfortunate thing is because of the way serverless uses the globs I can't exclude everything and then only include a few select items.  That would be more future proof but at quick glance this did the trick for now.